.github/workflows/release.yml

```diff
@@ -48,7 +63,7 @@ jobs:
           if [ -n "$BINARY_PATH" ]; then
             BRANCH_NAME=$(echo "${{ github.ref }}" | sed 's/refs\/heads\///')
             ARCHITECTURE=$(uname -m)
-            BINARY_NAME="cord-${BRANCH_NAME}-${ARCHITECTURE}-ubuntu-22.04"
+            BINARY_NAME="cord-${BRANCH_NAME}-${ARCHITECTURE}"
```

Comment:
Having different names for the binaries will break the tooling layer. Rather than building distribution-specific binaries, we should release distribution-specific packages. That workflow can be moved to another repo, as the package releases only need to happen after we tag a release candidate.

Reply:
This specific naming solution initially came from Amar. When a user downloads the binary our documentation force them to rename it directly to cord therefore tooling will keep working.

Reply:
That was long back. Since we are planning to move release activities to a separate repo, we don't need to make this change here. Safe to close this PR.